tests/test_room_api.py

```diff
@@ -173,6 +173,21 @@ def test_invalid_component(self, room_api: RoomApi):
             room_api.delete_component(room.id, "invalid_id")
 
 
+class TestHLSSubscribe:
+    def test_valid_subscription(self, room_api: RoomApi):
+        _, room = room_api.create_room(video_codec=CODEC_H264)
+        _ = room_api.add_component(
+            room.id, options=ComponentOptionsHLS(subscribe_mode="manual")
+        )
+        assert room_api.hls_subscribe(room.id, ["track-id"]) is None
+
+    def test_invalid_subscription(self, room_api: RoomApi):
+        _, room = room_api.create_room(video_codec=CODEC_H264)
+        _ = room_api.add_component(room.id, options=HLS_OPTIONS)
+        with pytest.raises(BadRequestException):
+            room_api.hls_subscribe(room.id, ["track-id"])
```

Comment:
Shouldn't we test all cases that we test in `elixir_server_sdk`. There we will have 5 cases, why don't we have them also here?

Reply:
I think we have discussed this previously, and we agreed that we don't necessarily need to test everything in both SDK's (we assume that it works in Elixir and it should here as well).
This approach has its pitfalls so we can reconsider if you think its appropriate.

Reply:
I agree with @roznawsk. This SDK is entirely generated so in my opinion it's pointless to test everything, for that we have `elixir_server_sdk`.